x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_form_text_area.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiFormRow, EuiTextArea } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  useEditTransformFlyout,
+  type EditTransformHookTextInputSelectors,
+} from './use_edit_transform_flyout';
+
+function capitalizeFirstLetter(str: string) {
```

Comment:
It would be great if you could move this to a shared utility file since it's now copied across `edit_transform_flyout_form_text_area.tsx` and `edit_transform_flyout_form_text_input.tsx`. Because it's not used outside the scope of these components you could just create a `capitalize_first_letter.ts` file in the same directory and move it there.